src/components/NavigationDrawer.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import { StackNavigationProp } from '@react-navigation/stack'
+import { StyleSheet, View, Image, TouchableOpacity } from 'react-native'
+import BottomSheet from 'reanimated-bottom-sheet'
+import Icon from 'react-native-vector-icons/FontAwesome5'
+
+import logo from '../../assets/logo.png'
+
+type NavigationBarProps = {
+    navigation: StackNavigationProp<>;
+  }
+
+const styles = StyleSheet.create( {
```

Comment:
@Harjot1Singh we should start thinking about theming. I want dark mode ;). This would be a good opportunity to try and have the same / similar mechanism that we use for desktop, or if that is not desirable, come up with a new mechanism that can go back into desktop

Reply:
I’d also want one too, we can tackle in separate issue 😄

Reply:
#58 will take care of this. Comment there for suggestions.